cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/worker/TaskWorkerTwillRunnable.java

```diff
@@ -99,6 +100,7 @@ protected void configure() {
         }
       });
       modules.add(new RemoteLogAppenderModule());
+      modules.add(new AuthenticationContextModules().getProgramContainerModule());
```

Comment:
I don't think ProgramContainerModule is the correct one. I believe it's used when running within Dataproc

Reply:
changed it to master module